libopenage/pathfinding/cost_field.cpp

```diff
@@ -54,6 +55,28 @@ void CostField::set_costs(std::vector<cost_t> &&cells, const time::time_t &valid
 
 	this->cells = std::move(cells);
 	this->valid_until = valid_until;
+	this->cell_cost_history.set_insert_range(valid_until, this->cells.begin(), this->cells.end());
+}
+
+bool CostField::stamp(size_t idx, cost_t cost, const time::time_t &stamped_at) {
+	if (this->cost_stamps[idx].has_value()) return false;
```

Comment:
Please always make if clauses multi-line for consistent code style.

```suggestion
	if (this->cost_stamps[idx].has_value()) {
	    return false;
	}
```